src/Template/Layout/styles/_elements.scss

```diff
@@ -294,3 +294,106 @@ a.close-button {
         transform: rotate(359deg)
     }
 }
+
+// create-new-object
+
+.upload-files {
```

Comment:
This should not go in `elements.scss`: the file contains the recurring UI patterns (tag, columns, boxes, loader...).